src/app/rating/rating-detail/rating-detail.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import {AuthenticationBasicService} from '../../login-basic/authentication-basic.service';
+import {Rating} from '../rating';
+import {RatingService} from '../rating.service';
+
+@Component({
+  selector: 'app-rating-detail',
+  templateUrl: './rating-detail.component.html',
+  styleUrls: ['./rating-detail.component.css']
+})
+export class RatingDetailComponent implements OnInit {
+  public rating: Rating = new Rating();
+
+  constructor(private route: ActivatedRoute,
+              private ratingService: RatingService,
+              private authenticationService: AuthenticationBasicService){
+  }
+
+  ngOnInit(): void {
+    const id = this.route.snapshot.paramMap.get('id');
+    this.ratingService.getResource(id).subscribe(
+       rating => {this.rating = rating;
```

Comment:
You are not getting the Resource and the author of the Rating so you can't show it in the HTML. This is making the page crash and the Rating is not shown.